appveyor.yml

```diff
@@ -0,0 +1,54 @@
+image: Visual Studio 2017
+
+# version format
+version: 0.0.{build}
+
+# UMBRACO_PACKAGE_PRERELEASE_SUFFIX if a rtm release build this should be blank, otherwise if empty will default to alpha
+# example UMBRACO_PACKAGE_PRERELEASE_SUFFIX=beta
+init:
+  - set UMBRACO_PACKAGE_PRERELEASE_SUFFIX=
+
+cache:
+  - src\packages -> **\packages.config  # preserve "packages" directory in the root of build folder but will reset it if packages.config is modified
+
+before_build:
+  - nuget restore src
+
+build_script:
+  - build-appveyor.cmd
+
+artifacts:
+  - path: artifacts\*.nupkg
+  - path: artifacts\*.zip
+
+deploy:
+    # MyGet Deployment for builds & releases
+  - provider: NuGet
+    server: https://www.myget.org/F/byte5digital/api/v2/package
+    symbol_server: https://www.myget.org/F/byte5digital/symbols/api/v2/package
```

Comment:
We set-up an [Umbraco community MyGet feed](https://www.myget.org/gallery/umbraco-packages), (it's not just UMCO packages, Jeavon and others use it too)

    server: https://www.myget.org/F/umbraco-packages/api/v2/package
    symbol_server: https://www.myget.org/F/umbraco-packages/symbols/api/v2/package

Reply:
That's really cool. IMHO we should use this feed :)